config.go

```diff
@@ -52,6 +53,30 @@ func NewFromPaths(paths []string) (viperChannel chan *viper.Viper, errChannel ch
 	return New(vipers)
 }
 
+/// NewFromPathsAndGlob takes as input an array of paths and a glob creates a
+/// ViperPit that monitors each available file matching the glob pattern at those
+/// paths.
+func NewFromPathsAndGlob(paths []string, glob string) (viperChannel chan *viper.Viper, errChannel chan error) {
+	// Make the array that'll store all our viper instances
+	var vipers []*viper.Viper
+
+	// Each viper instance looks at the given path for a file named
+	// config with any supported extension
+	for _, path := range paths {
+		matches, err := filepath.Glob(filepath.Join(path, glob))
+		if err != nil {
+			continue
```

Comment:
I'm not sure I like the quiet error/continue here, I can see it easily leading to confusion where users think the library is broken when, in fact, there's an fs error here that's quietly making the whole thing fail.

However, this isn't your fault, I think it brings up a larger issue of scalability and usability that the constructor assumes it can always make two channels and return them without another error. There will have to be a larger refactor to make handling this error here more easy.

For now, I'll merge this, but I expect it'll be refactored shortly so that you can call `New(...)` without any viper instances, and add others later. This way we can call `New()`, fetch the viper and error channels, and feed the `filepath.Glob` errors into the error channel as we come across them.